src/ui/style.css

```
@@ -1647,12 +1647,78 @@ body.cards-resizing {
+}
+/* The picker keeps a real minimum: with min-width 0 it collapsed to its arrow whenever the status
+   text competed for the row, leaving a dropdown whose own value was invisible. It still grows into
+   spare space, but never below a width that shows a file name. */
+.fileedit-pick { flex: 1 1 auto; min-width: 9ch; }
+
+/* Unsaved work, marked the way a modified editor tab is: a dot on the Save button. The button stays
+   readable on its own, so the dot is an addition rather than the only signal. */
+.fm-editor-save.dirty::after {
+    content: "";
+    display: inline-block; width: 6px; height: 6px; margin-left: 6px;
+    border-radius: 50%; background: currentColor; vertical-align: middle;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)